src/app/data/components/data-browser/data-browser.component.ts

```diff
@@ -544,6 +550,17 @@ export class DataBrowserComponent implements OnInit, OnDestroy {
     return false;
   }
 
+  _commentsCreateSeachGroup(item: SearchGroupItem): FormGroup {
```

Comment:
shouldn'T this function be in the commentsFilter class?

Reply:
Forgot to delete it from the data browser component.